### ISSUE TITLE
ERROR in ./index.js (./playground/examplesLoader.js!./index.js) 2:8

### ISSUE THREAD

**Issue by MohoiNad** [gabrielbull/react-desktop#140]
>Module parse failed: Expecting Unicode escape sequence \uXXXX (2:8)
You may need an appropriate loader to handle this file type.
| let data = {};
 > import \examples\macOs\advanced\pinDialog from '.\examples\macOs\advanced\pinDialog';
| data['\examples\macOs\advanced\pinDialog'] = \examples\macOs\advanced\pinDialog;
| import \examples\macOs\components\box from '.\examples\macOs\components\box';
 @ ./playground/playground.js 21:30-65
 @ ./playground/index.js
 @ multi (webpack)-dev-server/client?http://localhost:3001 ./playground/index.js

Child html-webpack-plugin for "index.html":
     1 asset
    Entrypoint undefined = index.html
    [./node_modules/html-webpack-plugin/lib/loader.js!./node_modules/html-webpack-plugin/default_index.ejs] 376 bytes {0} [built]
    [./node_modules/lodash/lodash.js] 527 KiB {0} [built]
    [./node_modules/webpack/buildin/global.js] (webpack)/buildin/global.js 472 bytes {0} [built]
    [./node_modules/webpack/buildin/module.js] (webpack)/buildin/module.js 497 bytes {0} [built]
i ｢wdm｣: Failed to compile.

Basically i just install package and then trying to <code>npm run playground</code>

I'm on Windows 10.

**Comment by davej** [comment]
Can't reproduce. Closing for the moment. If you can provide a reproducible example then I'll reopen.
